### ISSUE TITLE
Fails when using many slots

### ISSUE THREAD

**Issue by pyhys** [rtlabs-com/p-net#42]
If setting
`#define PNET_MAX_MODULES  17 `
(and modify the GSDML file accordingly) it is still only possible to use 9 input+output modules with the sample app.

When using slot 1-9 in Codesys everything works fine, but when using slot 1-10 this is the output:
```
Parameter write call-back. AREP: 1 API: 0 Slot: 9 Subslot: 1 Index: 123 Sequence: 18 Length: 4
  Bytes: 00 00 00 00 
Parameter write call-back. AREP: 1 API: 0 Slot: 9 Subslot: 1 Index: 124 Sequence: 19 Length: 4
  Bytes: 00 00 00 00 
Parameter write call-back. AREP: 1 API: 0 Slot: 10 Subslot: 1 Index: 123 Sequence: 20 Length: 4
  Bytes: 00 00 00 00 
Parameter write call-back. AREP: 1 API: 0 Slot: 10 Subslot: 1 Index: 124 Sequence: 21 Length: 4
  Bytes: 00 00 00 00 
[ERROR] CMRPC(2498): Unknown packet_type 1
[ERROR] CMRPC(2498): Unknown packet_type 1
Callback on event PNET_EVENT_ABORT. Error class: 253 Error code: 6
Aborting the application
```

**Comment by pyhys** [comment]
Changing the log level shows:

```
Parameter write call-back. AREP: 1 API: 0 Slot: 9 Subslot: 1 Index: 123 Sequence: 18 Length: 4
  Bytes: 00 00 00 00 
Parameter write call-back. AREP: 1 API: 0 Slot: 9 Subslot: 1 Index: 124 Sequence: 19 Length: 4
  Bytes: 00 00 00 00 
Parameter write call-back. AREP: 1 API: 0 Slot: 10 Subslot: 1 Index: 123 Sequence: 20 Length: 4
  Bytes: 00 00 00 00 
Parameter write call-back. AREP: 1 API: 0 Slot: 10 Subslot: 1 Index: 124 Sequence: 21 Length: 4
  Bytes: 00 00 00 00 
[DEBUG] BW(152): End of buffer reached
[DEBUG] BW(152): End of buffer reached
[DEBUG] BW(152): End of buffer reached
[DEBUG] BW(152): End of buffer reached
[DEBUG] BW(152): End of buffer reached
[DEBUG] BW(152): End of buffer reached
[DEBUG] BW(152): End of buffer reached
[DEBUG] BW(152): End of buffer reached
[DEBUG] BR(77): End of buffer reached
[ERROR] CMRPC(2498): Unknown packet_type 1
```
This is because the output buffer is full. It is implemented as 1500 bytes. Response message fragmentation is not yet implemented.

This is a duplicated of #33

**Comment by pyhys** [comment]
We have implemented a fix for this, see #62
